### ISSUE TITLE
Session error again

### ISSUE THREAD

**Issue by trantrung22** [gatecrasher777/ytcog#67]
It's unable to get session and show message "Session error". 
Seem like ytb have changed something, pls check.

**Comment by gatecrasher777** [comment]
Thanks for letting me know. Will do so this weekend. Will also try to fix up other reported bugs.

**Comment by trantrung22** [comment]
Hello, do you have a solution yet?

**Comment by gatecrasher777** [comment]
No, I pretty much gave up trying to keep up with YT increasing the barriers. Not just with their tweaks to the player code but barring all downloads outside of the own apps and websites. 

I have been trying the past few days to get everything back up and running on one of my YT projects, If I am successful, I will update this project with the solutions I discover.
